includes/admin/class-sensei-setup-wizard.php

```diff
@@ -93,7 +93,7 @@ public function __construct() {
 			add_action( 'current_screen', [ $this, 'remove_notices_from_setup_wizard' ] );
 			add_action( 'admin_notices', [ $this, 'setup_wizard_notice' ] );
 			add_action( 'admin_init', [ $this, 'skip_setup_wizard' ] );
-			add_action( 'admin_init', [ $this, 'activation_redirect' ] );
+			add_action( 'current_screen', [ $this, 'activation_redirect' ] );
```

Comment:
It's changed because now we want to use the `get_current_screen`, and it's not ready in the `admin_init` hook.